LICENSE

```diff
@@ -0,0 +1,37 @@
+MIT License
+
+Copyright (c) 2023 Michał Blinkiewicz
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS," WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT, OR OTHERWISE, ARISING FROM,
+OUT OF, OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+==============================================================================
+                            Important Licensing Note
+==============================================================================
+
+Please be aware that this repository contains a directory named "patches" with
+binary data files. These files are subject to a separate licensing arrangement
+and are marked as "All Rights Reserved." This means they are protected by
+copyright, and no specific license information is provided for them. Users
+should assume that they have no rights or permissions to use, modify, or
```

Comment:
This means that without express "permission to use [the patches]", people cannot install your bundle of hacks.

Also the phrasing - "should assume that" - is odd. 
I suggest: "Users are not allowed to modify or distribute these files without explicit permission from the copyright holder."

Beware, tho, IANAL

Reply:
Thanks for the suggestion, I've just applied it.